.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,33 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+  
+  backend-build:
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./backend
+    environment: development 
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image for WebApi
+      run: docker build . --file ./src/WebApi/Dockerfile --tag web-api:$(date +%s)
+      
+  frontend-build:
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./frontend
+    environment: development 
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image for WebApi
```

Comment:
Tutaj jest napisane, że tylko budujesz, ale Ty uruchamiasz ten kontener, co później skutkuje tym, że on nigdy nie skończy działania (dlatego trzeba było dać cancel po parunastu minutach):
```
Successfully built 6dac07245639
Successfully tagged frontend_developement:latest
Image for service developement was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
Creating web-client ... 
Creating web-client ... done
Attaching to web-client
web-client      | 
web-client      | > partyklinest-frontend@0.1.0 start /app
web-client      | > react-scripts start
web-client      | 
web-client      | (node:26) [DEP_WEBPACK_DEV_SERVER_ON_AFTER_SETUP_MIDDLEWARE] DeprecationWarning: 'onAfterSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
web-client      | (node:26) [DEP_WEBPACK_DEV_SERVER_ON_BEFORE_SETUP_MIDDLEWARE] DeprecationWarning: 'onBeforeSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
web-client      | Starting the development server...
web-client      | 
web-client      | Compiled successfully!
web-client      | 
web-client      | You can now view partyklinest-frontend in the browser.
web-client      | 
web-client      |   Local:            http://localhost:3000
web-client      |   On Your Network:  http://172.18.0.2:3000/
web-client      |
```

Reply:
No tak. Działa dobrze ale potrzebujemy czego innego.